iri-commons/src/main/java/no/sikt/nva/iri/PublicationChannelId.java

```diff
@@ -0,0 +1,136 @@
+package no.sikt.nva.iri;
+
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import java.net.URI;
+import java.time.DateTimeException;
+import java.time.Year;
+import java.util.UUID;
+
+import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
+
+public interface PublicationChannelId {
```

Comment:
90% of this interface is common validation rules for PublicationChannelId, in addition to 4 public getters(). What about moving all the validation to PublicationChannelIdValidator? Separation of concerns :)